src/smart-components/role/RolePermissionsTable.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Access, Role } from '../../redux/reducers/role-reducer';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+
+interface RolePermissionsTableProps {
+  viewedRole?: Role;
+}
+
+export const RolePermissionsTable: React.FunctionComponent<RolePermissionsTableProps> = ({ viewedRole }) => {
+  const COLUMNS: string[] = ['Application', 'Resource type', 'Operation'];
+
+  const transformRow = (access: Access) => {
+    const [application, resource, operation] = access.permission.split(':');
+    console.log(application, resource, operation);
+    return [application, resource, operation];
+  };
```

Comment:
```suggestion
  const transformRow = (access: Access) => {
    const [application, resource, operation] = access.permission.split(':');
    return [application, resource, operation];
  };
```
We could move this one out of the component, just not to reinitialize it on every render.